spec/std/enumerable_spec.cr

```diff
@@ -393,6 +394,33 @@ describe "Enumerable" do
     end
   end
 
+  describe "each_step" do
+    it_iterates "yields every 2nd element", %w[a c e], %w[a b c d e f].each_step(2)
+    it_iterates "accepts an optional offset parameter", %w[b d f], %w[a b c d e f].each_step(2, offset: 1)
+    it_iterates "accepts an optional offset parameter of 0", %w[a c e], %w[a b c d e f].each_step(2, offset: 0)
```

Comment:
I would like extra spec(s) where `n > 0 && (n <= offset < size)`, which would skip some elements at the beginning